src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Color[] colors = Color.values();
+        Random random = new Random();
```

Comment:
Think about which variables should be local in the method and which should be class-level. new Random() object will be created on each getRandomColor() call. How can you avoid this?

Reply:
I dont know. I have no clue. Absolutely